src/plugin/callback_msg.rs

```diff
@@ -80,6 +80,7 @@ pub(super) struct MsgToExtSupport {
 
 #[derive(Debug, Serialize)]
 struct PluginSignReq {
+    uuid: Vec<u8>,
     peer_id: String,
```

Comment:
Remove peer_id then?

Reply:
I thought about it, the `uuid` needs to be included in the msg to ensure that the uuid is valid and not filled in arbitrarily.